tests/platform_tests/templates/expect_boot_messages

```diff
@@ -1,4 +1,7 @@
 r, ".* NOTICE admin: (Stopping|Stopped|Starting|Started).* (?!syncd|swss|gbsyncd).* service.*"
 r, ".* NOTICE admin: Stopped.* swss.* service.*"
 r, ".* NOTICE root: (Stopping|Stopped|Starting|Started).* (swss|syncd|gbsyncd).* service.*"
-r, ".* NOTICE root: WARMBOOT_FINALIZER.*"
\ No newline at end of file
+r, ".* NOTICE root: WARMBOOT_FINALIZER.*"
+r, ".*\|c\|SAI_OBJECT_TYPE_SWITCH.*"
```

Comment:
Just curious - will these regexes work alike for Broadcom, Mellanox, KVM, 201911 and master?

Reply:
I updated the default route regex so that it works with earlier branches. Then these regexes should not be affected by branches or platforms in the current stage or the foreseeable future.